app/views/static_pages/home/_curriculum.html.erb

```diff
@@ -9,14 +9,15 @@
         <div class="row curriculum-details-grid">
 
           <% available_courses.each do |course| %>
-          <div class="curriculum-details-tile align-items-center">
-            <%= image_tag course[:badge_image_url], alt: course[:badge_alt_text] %>
-            <h4 class="bold text-uppercase">
-              <%= course[:title] %>
-            </h4>
-          </div>
+            <div class="curriculum-details-tile align-items-center">
```

Comment:
Since we are wrapping the image and course title in a link now we can get rid this div that was previously wrapping these elements. If we do that and add the classes on this div we can remove the new `.course-tile-path` css you've added as the other classes will take care of that. This is what the final code would look like:

```
<%= link_to course[:path], class: "curriculum-details-tile align-items-center" do %>  
  <%= image_tag course[:badge_image_url], alt: course[:badge_alt_text] %>
   <h4 class="bold text-uppercase">
      <%= course[:title] %>
    </h4>
<% end %>
```